src/store/api/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as contentful } from "./contentful";
+export * from "./contentful";
```

Comment:
I am not a fan of the double exports. But I also  don't know a better format.

Reply:
yeah, this bothers me a lot too

Reply:
```
export const { useAllCommitsQuery, useMembersQuery } = github;
export const github;
```

```
export * from "./github";
```

Maybe by exporting `github` also as const? you are basicly turning the default export in the property with the same name

Reply:
even better:

```
export const github = createApi({
  reducerPath: 'github',
  baseQuery: fakeBaseQuery(),
  endpoints: build => ({
    allCommits: build.query({ queryFn: allCommits }),
    members: build.query({ queryFn: allMembers })
  })
});

export const { useAllCommitsQuery, useMembersQuery } = github;
```

Reply:
i like the idea, i'll implement that!